tools/autograd/gen_python_functions.py

```diff
@@ -763,7 +763,8 @@ def is_arg_smaller(t1: Type, t2: Type) -> bool:
         # discussed why it is important to prioritize int/int? over int[]
```

Comment:
Since there are now a lot of special cases it seems like it would be better to put the comment directly above the relevant line

Reply:
Thanks for recommending and it is done: https://github.com/pytorch/pytorch/pull/54617/files#  
(Sorry for the confusion, there are to PR's regarding this task, for some reason and #54617 is the one we intend to merge)